src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/roles/edit/details/main.js

```diff
@@ -226,6 +235,18 @@ define([
             this.sandbox.start(this.$el);
         },
 
+        selectAllOrNone: function () {
+            if(noneSelected) {
```

Comment:
Should be `!!noneSelected`. Actually, when thinking of it, wouldn't `someSelected` be the better name?